dluhc-component-library/src/components/siteSelectionForm/components/CheckAnswers.tsx

```diff
@@ -0,0 +1,9 @@
+const CheckAnswers = () => {

```

Comment:
This page isn't used yet so we don't need it in this PR. ( I know its a WIP)

I think for the first PR we can just have the submit button on the final question. That lets us split the check answers page into another PR keeping this small.